src/main/java/TaskManager.java

```diff
@@ -0,0 +1,99 @@
+public class TaskManager {
+    private Task[] tasks = new Task[100];
+    private int numTask = 0;
+
+    public static final String BORDER = "_________________________________________\n";
+    public static final String TASK_ADDED_MESSAGE = BORDER + "The task has been added: ";
+    public static final String INVALID_TASK_NUMBER = "Invalid task number";
+    public static final String TASK_COMPLETE_MESSAGE = "Congrats on finishing a task! Have a cookie!";
+    public static final String UNKNOWN_COMMAND_MESSAGE = "I'm sorry, I didn't understand.";
+    public static final String NO_TASKS_MESSAGE = "No Tasks";
+
+    private static final String GREETING = "****************************\n"
+            + "*  ____             ____   *\n"
+            + "* |  _ \\    ____   |  _ \\  *\n"
+            + "* | |_| |  / __ \\  | |_| | *\n"
+            + "* |  _  | | |  | | |  _  | *\n"
+            + "* | |_| | | |__| | | |_| | *\n"
+            + "* |____/   \\____/  |____/  *\n"
+            + "****************************\n"
+            + BORDER
+            + "Have no fear, Bob is here!\n"
+            + "What is it that you require?\n"
+            + BORDER;
+
+    public static final String EXIT_MESSAGE = "Bye. Have a nice day!\n";
+
+    public TaskManager() {
+    }
+
+    public void greet() {
+        System.out.println(GREETING);
+    }
+
+    public void exitMessage() {
+        System.out.println(BORDER + EXIT_MESSAGE + BORDER);
+    }
+
+    public void addTodoTask(String description) {
```

Comment:
Adding some comments to these methods will improve clarity as they aren't exactly trivial.